bcdaworker/repository/postgres/repository_test.go

```diff
@@ -180,3 +189,22 @@ func assertJobsEqual(assert *assert.Assertions, expected, actual models.Job) {
 	expected.TransactionTime, actual.TransactionTime = expected.TransactionTime.UTC(), actual.TransactionTime.UTC()
 	assert.Equal(expected, actual)
 }
+
+// cryptoRandInt31 generates a random int31 using crypto/rand
+func cryptoRandInt31() int32 {
+	n, err := rand.Int(rand.Reader, big.NewInt(1<<31))
+	if err != nil {
+		panic(err)
+	}
+	o, _ := safecast.ToInt32(n.Int64())
+	return o
+}
+
+// cryptoRandInt63 generates a random int63 using crypto/rand
+func cryptoRandInt63() int64 {
+	n, err := rand.Int(rand.Reader, big.NewInt(math.MaxInt64))
+	if err != nil {
+		panic(err)
+	}
+	return n.Int64()
+}
```

Comment:
Im seeing these functions in a few places.  Is there a good way to re-use these across packages?  Or is it preferred in Go to keep this at the package level?  Also a bit concerned that sonarqube might flag these as duplicate.

Reply:
We have this [testutils](https://github.com/CMSgov/bcda-app/tree/main/bcda/testUtils) which is good for generic functions that do not rely on any of our internal packages (cyclic imports everywhere), if moving these there wouldn't be too cumbersome?